__tests__/textalyze.test.js

```diff
@@ -1,4 +1,6 @@
-const { itemCounts } = require('../textalyze');
+const {
+  itemCounts, stringCharsToArray, sanitize, readFromFile,
+} = require('../textalyze');
```

Comment:
I like that you're listening to the linter! :)

Reply:
:)